docker/test/integration/minifi/validators/NoContentCheckFileNumberValidator.py

```diff
@@ -0,0 +1,25 @@
+import logging
+import os
+
+from .FileOutputValidator import FileOutputValidator
+
+
+class NoContentCheckFileNumberValidator(FileOutputValidator):
+    """
+    Validates the content of a single file in the given directory.
+    """
+
```

Comment:
This comment disagrees with the code

Reply:
Fixed in f1c9709ef6a9a2ea4ff8cf3e9e257123e8cd12dc